_upload_data.py

```diff
@@ -0,0 +1,72 @@
+import click
+import os
```

Comment:
I would say try using `pathlib` instead of `os.path`. It is easier to read and make your life so much better :grin: 

(I know, alex will say I have TOCs)

Reply:
I am just wondering which meaning of TOC you have in mind:

https://acronyms.thefreedictionary.com/TOC

maybe this one: Taste of chaos? :-p

Reply:
Look for french ones :)

Reply:
Hm. Why is it easier to read? And why is it better? Convince me ;-)

Reply:
It is easier to read because once you have done `local_path = Path(local_path)`, you can `join` simply with `  local_path / project_type`, you can list files recursively with `local_path.rglob('*')` you can get parent directory with `local_path.parent` and you can get relative path with `local_path.relative_to`.

I feel it is easier because you simply have an object and you don't need to call imbricated functions.

used_dir = local_path / project_type
LOCAL_PATH, project_type + '.tar.gz'